types/core/attributes.go

```diff
@@ -34,6 +34,8 @@ type ActionAttributes interface {
 // attribute type has to implement.
 type ForwardingAttributes interface {
 	proto.Message
-	// Returns the destination chain identifier.
+	// CounterpartyID returns the destination chain identifier.
 	CounterpartyID() string
+	// Validate the attributes.
```

Comment:
nit: just for consistency with the change you made in the `CounterpartyID` docstring
```suggestion
	// Validate returns an error if the attributes are not valid.
```